editor.planx.uk/src/@planx/graph/index.ts

```diff
@@ -507,58 +507,115 @@ export const sortIdsDepthFirst =
 export const formatOps = (graph: Graph, ops: Array<OT.Op>): string[] => {
   const output: string[] = [];
 
+  // Only show full change description for simple props, omit complex or long ones like `moreInfo`, `fileTypes`, etc
+  const allowProps = ["title", "text", "fn", "val"];
```

Comment:
I think this is the best MVP solution for easy-to-digest edit summaries & fixes previously pointed out bugs.

For example, "Updated UploadAndLabel fileTypes [object object]" &rarr; "Updated UploadAndLabel fileTypes". See new `formatOps()` test cases for more.

Longterm, we still may want to explore _each_ component type managing its' own formatting rules based on possible props, but that's going to be a quite large undertaking!

Reply:
I think this is a perfect MVP approach - if a user ever asks for more detail here we can add it as a ticket and measure it on it's own cost/reward ratio.